app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryResultsRow.tsx

```diff
@@ -23,13 +23,20 @@ export const ListingDetailsLotteryResultsRow = ({ bucket }: LotteryResultsRowPro
           {bucket.preferenceName === "generalLottery"
             ? t("lottery.generalPool")
             : defaultIfNotTranslated(
-                `listings.lotteryPreference.${bucket.preferenceName}.title`,
-                bucket.preferenceName
+                `listings.lotteryPreference.${bucket.preferenceName
+                  .replace("Tier 1 ", "")
+                  .replace("Tier 2 ", "")}.title`,
+                bucket.preferenceName.replace("Tier 1 ", "").replace("Tier 2 ", "")
               )}
         </Heading>
+        {bucket.preferenceName.includes("Tier 1")
```

Comment:
question: Are we sure this condition is sufficient? There won't new preferences in the near future that also have `"Tier 1"` in their string name?

Reply:
I was approaching this as a stopgap design, anticipating a much bigger redesign later before any new preferences get added, but maybe that's a risky assumption

Maybe I can branch off the way I did with the pr for before the lottery number is entered, and make a component that will only be used for educators?

Reply:
I haven't looked at the other PR, but componentization is always good!